recipe/meta.yaml

```
@@ -107,24 +92,25 @@ requirements:
     - libspatialindex
     - libspatialite
     # For PostgreSQL support.
-    - postgresql
+    - postgresql >=14,<15
     - psycopg2
+    # Point Clouds
+    - pdal
+    - laz-perf
   run:
     - python
     - icu
     - libzip
     - libtasn1  # [osx]
-    - qt
+    - qt-main
     - pyqt
     - qtwebkit
     - qscintilla2
     - qca
     - qjson
     - qwt
-    - qwtpolar
     - qtkeychain
-    - qtlocation
-    - qtserialport  # [unix]
+    - libgdal
     - gdal
```

> Should work with local pinning, I think you need to have:
> 
> ```
> - {{ pin_compatible('gdal', max_pin='x.x') }}
> - {{ pin_compatible('libgdal', max_pin='x.x') }}
> ```
> 
> In the meta.yaml (instead of `gdal` and `libgdal`), but I'm no expert. @weiji14 may be able to advise?

Yes, set in in the runtime dependencies. I don't think you need to declare `libgdal` as a dependency if `gdal` is pinned, but won't hurt I guess.

```suggestion
    - {{ pin_compatible('libgdal', max_pin='x.x') }}
    - {{ pin_compatible('gdal', max_pin='x.x') }}
```